### DISCUSSION TITLE
Problem on fresh Windows 11 N with media extension pack installed

### DISCUSSION THREAD

**Discussion by MeierAlex** [emgucv/emgucv#665] [Q&A]
Hallo,

I have Problems with a software using this library running on Windows 11 N with already installed media extension pack. The software simply does not start and I dont get any output on the commandline. Here is a[ link ](https://github.com/sn4k3/UVtools/issues/356#issuecomment-985687428)to the issue on the repository of the software I an trying to use.

Best regards 
Alex

**Comment by emgucv** [comment]
Please follow this instructions:
https://www.emgu.com/wiki/index.php/Download_And_Installation#Are_you_missing_any_dependency.3F
and upload the resulting ".dwi" from dependency walker. 

We will be able to take a look once we have the ".dwi" file.

**Reply by sn4k3** [reply]
Problem was fixed by the OP:

`Fixed the problem - you have to install the media extensions via windows by "Add optional feature" -> "Media Extensions"`
